stable/prometheus-mongodb-exporter/README.md

```diff
@@ -62,7 +62,11 @@ podAnnotations:
 | `serviceMonitor.namespace` | The namespace where the Prometheus Operator is deployed | `` |
 | `serviceMonitor.additionalLabels` | Additional labels to add to the ServiceMonitor | `{}` |
 | `tolerations` | List of node taints to tolerate  | `[]` |
-
+| `tolerations` | List of node taints to tolerate  | `[]` |
+| `cacert.enabled` | Enable TLS  | `false` |
+| `cacert.mountPath` | Mount path for certification | `/var/mongodb_exporter/config` |
+| `cacert.content` | Contents of where the certification is mounted. | `""` |
+| `cacert.name` | Name of volume | `""` |
```

Comment:
Unused variable. Probably can be removed as we can just use `{{ include "prometheus-mongodb-exporter.fullname" . }}-cacert` like you do